src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/MarkupCompiler/PathInternal.cs

```diff
@@ -0,0 +1,382 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//---------------------------------------------------------------------------
+//
+// Description:
+//   Returns a relative path from one path to another.  
```

Comment:
Could you comment and add where this source came from (repo, sha etc.?) so that future maintenance would be easier?

Reply:
> //   Ported to .NET Framework from the GetRelativePath implementation in:
> //
> //   https://github.com/dotnet/corefx/blob/b123ba4b9107c73cbc02010dc1ee78eb8ffccb93/src/Common/src/CoreLib/System/IO/Path.cs
> //
> //   This ported method ('GetRelativePath') is only called by the .NET Framework 
> //   version of PresentationBuildTasks.dll that ships with the Microsoft.NET.Sdk.WindowsDesktop 
> //   SDK as part of .NET 5.0.